requirements.txt

```diff
@@ -0,0 +1,5 @@
+asgiref==3.5.2
+Django==4.0.5
+sqlparse==0.4.2
+tzdata==2022.1
+
```

Comment:
@parthasarathi9438 Add other requirements packages and dependencies

Reply:
resolved